baseobject/mock_package/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
```

Comment:
Eventually I'd like for us to come up with somewhere else for this. But as we've talked about, the best location is not clear yet. So let's keep it here in V0.1.0 and revisit in V0.2.0.

One option is to move it as a submodule under "tests", since it isn't really user functionality. But as you've pointed out there may be some documentation/example benefit of it. So I think maybe a better option is to have a "template" mock package in its own repository that we can use in the tests and point to as an example of how you can use BaseObject.